sauna/plugins/ext/tcp.py

```diff
@@ -0,0 +1,44 @@
+import socket
+
+from sauna.plugins import (Plugin, bytes_to_human, human_to_bytes,
+                           PluginRegister)
+
+my_plugin = PluginRegister('TCP')
+
+
+@my_plugin.plugin()
+class Tcp(Plugin):
+
+    def __init__(self, config):
+        super().__init__(config)
+        self.config = {
+            'host': config.get('host', 'localhost'),
+            'port': config.get('port', 11211),
+            'timeout': config.get('timeout', 5)
+        }
+        self._stats = None
+
+    @my_plugin.check()
+    def request(self, check_config):
+        try:
+            with socket.create_connection((check_config['host'],
+                                           check_config['port']),
+                                          timeout=check_config['timeout']) as s:
+                if s is None:
```

Comment:
Is it better to set return in else than in the with ? I think they are equals, but why to use else ?